python/pyarrow/src/arrow/python/arrow_to_pandas.cc

```diff
@@ -176,6 +176,7 @@ static inline bool ListTypeSupported(const DataType& type) {
     case Type::DATE32:
     case Type::DATE64:
     case Type::STRUCT:
+    case Type::MAP:
```

Comment:
There didn't seem to be a strong reason to have this disabled. Users can do nested maps via (list of structs of maps)

Reply:
Agreed. Although it's interesting we check the value type for list, but not the fields of structs? I was thinking you should put `Type::MAP` down with `Type::LIST`, since it is a subclass of ListType, though since the immediate child is `Type::STRUCT` right now it won't do anything meaningful.